templates/node/package.json.twig

```diff
@@ -11,7 +11,8 @@
     "url": "{{ sdk.gitURL }}"
   },
   "devDependencies": {
-    "@types/node": "^18.16.1"
+    "@types/node": "^18.16.1",
+    "jest": "^29.7.0"
```

Comment:
Not a blocker but just noting Node actually has built in test runner for a while now. We could use it in the future

Reply:
Yeah, I saw that mentioned somewhere the other day but didn't look into it further; experimental in Node 18, stable since 20; I could look into migrating this if needed!

Point of concern, since I am not too familiar with distributing libraries like this: Wouldn't tests in the long term not also be run on older Node versions than 18 to ensure they are compatible with those versions? Or are there plans to drop anything below 18 anyway since 16 LTS has reached EOL earlier this year?

Reply:
The tests only have to work on CI and developer environment so that's fine. 
The generated SDK needs to support 16 and above. We're a little bit behind node EOL.

If you are interested to look into it that'd be awesome, but it's not a blocker :)

Reply:
'ello! I took a peek and for the most part, just dropping jest and adding `const { describe, test } = require("node:test");` should be sufficient. I did stumble over one issue though and that is mocking of modules. Specifically, there doesn't seem to be a sane way of mocking axios in this instance. After some research, I stumbled over https://github.com/nodejs/node/issues/51164 with still ongoing discussion on whether / how module level mocking is supported at the moment.

Given that there doesn't seem to be a documented example on how to properly achieve this, I think I'd rather hold off on this for now but I am down to possibly revisit this in the future!

Reply:
Makes sense, sounds like a good plan